ietf/search/views.py

```diff
@@ -4,18 +4,23 @@
 from wagtail.contrib.search_promotions.models import Query
 from wagtail.models import Page
 
+MAX_SEARCH_TERMS = 200
+
 
 def search(request):
     search_query = request.GET.get("query", None)
     page = request.GET.get("page", 1)
 
     # Search
-    if search_query and "\x00" not in search_query:
+    if (
+        search_query
+        and search_query.count(" ") > MAX_SEARCH_TERMS
+        or "\x00" in search_query
```

Comment:
This will be evaluated as (A and B) or C. The C clause is not guarded against search_query == None and `"\x00" in None` will fail.

Reply:
I think you want A and (B or C)

Reply:
My original code has `A and (B or C)` before `black` removed it.
Then I learned that,
```
>>> True and False or True
True
>>> True and True or False
True
>>> True and False or False
False
```
But interestingly `ruff` doesn't remove the brackets.
But I think having brackets are better. I'll add them.

Reply:
Fixed with https://github.com/ietf-tools/www/pull/552/commits/cb7bf78d0d49f2dfabe06c18d8be053e7e5e8604